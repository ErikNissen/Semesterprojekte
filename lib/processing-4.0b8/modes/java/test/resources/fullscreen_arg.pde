version https://git-lfs.github.com/spec/v1
oid sha256:984f66cc8a4159b64169a4f1544ebdd6dbaed0df03769cbc530fe1ad1d7fa473
size 21
