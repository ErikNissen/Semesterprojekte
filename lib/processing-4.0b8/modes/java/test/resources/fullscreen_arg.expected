version https://git-lfs.github.com/spec/v1
oid sha256:5f8ad7ff97bf5cfb1ee3e5a80d38d998bd178b2a79eb429652366be0df5076dd
size 803
